README.md

```diff
@@ -5,38 +5,67 @@
 
 ## Purpose
 
-Dextrose cli aims to provide a simple way to get visual snapshots of specified components across all react native platforms.
+Dextrose cli integrates into developer workflow by  providing visual snapshots of react native components described by storybook across all react native platforms.
```

Comment:
extra space between 'by' and 'providing'

Reply:
Could you maybe explain what Storybook is here? Or would we be assuming people using dextrose would know? Maybe we could at least add a link to the Storybook github repo? (https://github.com/storybooks/storybook)